src/autopas/utils/logging/IterationLogger.h

```diff
@@ -52,8 +53,9 @@ class IterationLogger {
 
  private:
   static inline std::string loggerName() { return "IterationLogger"; };
```

Comment:
suffix should go in here as well, in case we have two autopas instances on the same process